src/50_Sample_AMPs/Live_Blog.html

```diff
@@ -0,0 +1,129 @@
+<!---{
+  "preview": true,
+  "draft": true,
+  "experiment": true,
+  "component": "amp-live-list"
+  }--->
+<!--
+  #### Introduction
+
+This is a sample template for a live blogs in AMP. It demonstrates the usage of `amp-live-list` component which allows to create live blogs.
```

Comment:
s/a live/implementing live/
remove "It demonstrates the usage of `amp-live-list` component which allows to create live blogs."